pkg/action/install.go

```diff
@@ -564,6 +564,10 @@ func (i *Install) NameAndChart(args []string) (string, string, error) {
 	if base == "." || base == "" {
 		base = "chart"
 	}
+	// if present, strip out the file extension from the name
+	if idx := strings.Index(base, "."); idx != -1 {
```

Comment:
Are there any valid chart names that could have `.` in the name? I feel like there possibly could be and it would be better to use `strings.ReplaceAll(base, file path.Ext(base), "")`

Reply:
`path.Ext` only catches the first file extenion, but won't capture the `.tar.gz` case. I'm not sure if there's a way we can detect that without dropping down to `os.Stat` to check if the file handler is for a directory.

Reply:
Ok, sounds good.